packs/msn.yaml

```diff
@@ -0,0 +1,156 @@
+    name: msnrollingeyes
+  - src: http://www.msgweb.nl/emoticons/77_77.gif
+    name: msndrowsy
+  - src: http://www.msgweb.nl/emoticons/coffee.gif
+    name: msncoffee
+  - src: http://www.msgweb.nl/emoticons/thumbs_up.gif
+    name: msnthumbup
+  - src: http://www.msgweb.nl/emoticons/thumbs_down.gif
+    name: msnthumbdown
+  - src: http://www.msgweb.nl/emoticons/beer_mug.gif
+    name: msnbeer
+  - src: http://www.msgweb.nl/emoticons/martini.gif
+    name: msncocktail
+  - src: http://www.msgweb.nl/emoticons/girl.gif
+    name: msngirl
+  - src: http://www.msgweb.nl/emoticons/guy.gif
+    name: msnguy
+  - src: http://www.msgweb.nl/emoticons/guy_hug.gif
+    name: msnguyhug
+  - src: http://www.msgweb.nl/emoticons/girl_hug.gif
+    name: msngirlhug
+  - src: http://www.msgweb.nl/emoticons/bat.gif
+    name: msnbat
+  - src: http://www.msgweb.nl/emoticons/cake.gif
+    name: msncake
+  - src: http://www.msgweb.nl/emoticons/heart.gif
+    name: msnheart
+  - src: http://www.msgweb.nl/emoticons/broken_heart.gif
+    name: msnbrokenheart
+  - src: http://www.msgweb.nl/emoticons/kiss.gif
+    name: msnkiss
+  - src: http://www.msgweb.nl/emoticons/present.gif
+    name: msnpresent
+  - src: http://www.msgweb.nl/emoticons/rose.gif
+    name: msnrose
+  - src: http://www.msgweb.nl/emoticons/wilted_rose.gif
+    name: msnwiltedrose
+  - src: http://www.msgweb.nl/emoticons/camera.gif
+    name: msncamera
+  - src: http://www.msgweb.nl/emoticons/film.gif
+    name: msnfilm
+  - src: http://www.msgweb.nl/emoticons/cat.gif
+    name: msncat
+  - src: http://www.msgweb.nl/emoticons/dog.gif
+    name: msndog
+  - src: http://www.msgweb.nl/emoticons/phone.gif
+    name: msnphone
+  - src: http://www.msgweb.nl/emoticons/lightbulb.gif
+    name: msnlightbulb
+  - src: http://www.msgweb.nl/emoticons/note.gif
+    name: msnmusicnote
+  - src: http://www.msgweb.nl/emoticons/moon.gif
+    name: msnsleepingmoon
+  - src: http://www.msgweb.nl/emoticons/star.gif
+    name: msnstar
+  - src: http://www.msgweb.nl/emoticons/envelope.gif
+    name: msnenvelope
+  - src: http://www.msgweb.nl/emoticons/clock.gif
+    name: msnclock
+  - src: http://www.msgweb.nl/emoticons/messenger.gif
+    name: msnmsnmessengericon
+  - src: http://www.msgweb.nl/emoticons/53_53.gif
+    name: msnsnail
+  - src: http://www.msgweb.nl/emoticons/70_70.gif
+    name: msnblacksheep
+  - src: http://www.msgweb.nl/emoticons/55_55.gif
+    name: msnplate
+  - src: http://www.msgweb.nl/emoticons/56_56.gif
+    name: msn
+  - src: http://www.msgweb.nl/emoticons/57_57.gif
+    name: msnpizza
+  - src: http://www.msgweb.nl/emoticons/58_58.gif
+    name: msnsoccer
+  - src: http://www.msgweb.nl/emoticons/59_59.gif
+    name: msncar
+  - src: http://www.msgweb.nl/emoticons/60_60.gif
+    name: msnairplane
+  - src: http://www.msgweb.nl/emoticons/61_61.gif
+    name: msnumbrella
+  - src: http://www.msgweb.nl/emoticons/62_62.gif
+    name: msnıslandwithpalmtree
```

Comment:
> Error: custom emoji names can only contain lower case letters, numbers, dashes and underscores: msnıslandwithpalmtree

https://travis-ci.org/hugovk/emojipacks/builds/198613178#L964

That `ı` should be an `i`.